Chart.yaml

```diff
@@ -27,16 +27,19 @@ dependencies:
   - name: prometheus
     repository: https://kubernetes-charts.storage.googleapis.com
     version: 11.0.4
+    condition: prometheus.enabled
   - name: elasticsearch
     repository: https://helm.elastic.co
     version: 7.6.2
     condition: elasticsearch.enabled
   - name: kafka
     repository: https://charts.bitnami.com/bitnami
     version: 7.2.9
+    condition: kafka.enabled
   - name: grafana
     repository: https://kubernetes-charts.storage.googleapis.com
     version: 5.0.10
+    condition: grafana.enabled
 
 # A chart can be either an 'application' or a 'library' chart.
 #
```

Comment:
I think it'd be nice to update to version in this file with this change. To 0.2.0 probably :-)